api-schema/xdb.yaml

```diff
@@ -258,11 +258,9 @@ components:
     TimerCommand:
       type: object
       required:
-        - firingUnixTimestampSeconds
+        - delayInSeconds
```

Comment:
I think I like it. 
We used `firingUnixTimestampSeconds` since iWF -- but the absolute time value makes it a little hard for write unit tests.

Reply:
I think so. The client should just send a delay and the servere will transform the dely to an abosolute timestamp.